conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -153,6 +153,7 @@ def environment(self):
         env.append("CXXFLAGS", self._filter_list_empty_fields(self.cxxflags))
         env.append("CFLAGS", self._filter_list_empty_fields(self.cflags))
         env.append("LDFLAGS", self._filter_list_empty_fields(self.ldflags))
+        env.append_path("PKG_CONFIG_PATH", self._conanfile.generators_folder)
```

Comment:
```suggestion
        env.prepend_path("PKG_CONFIG_PATH", self._conanfile.generators_folder)
```

Shouldn't this path go first, so it has priority?

Reply:
```suggestion
        env.append_path("PKG_CONFIG_PATH", self._conanfile.generators_folder)
```
Yes... 🤦